api-reference/50 Common/Object Structures/positionConfig/collision/collision.md

```diff
@@ -5,48 +5,127 @@ type: String | Object
 ---
 ---
 ##### shortDescription
-Specifies how to move the widget if it overflows the screen.
+Specifies how to resolve collisions between the overlay and the [boundary](/Documentation/ApiReference/Common/Object_Structures/positionConfig/#boundary) elements.
 
 ---
-A string passed to this option should contain a collision handler name, or a pair of names separated by space. If the string consists of a single collision handler name, it is applied to both horizontal and vertical axes. If you pass a pair of names separated by space, the first collision handler will be applied to the horizontal axis, the second to the vertical axis.
+Collisions occur whenever the specified position causes the overlay element to exceed the boundaries.
```

Comment:
Do you mean:
Overlap occurs when the specified position causes an overlay element to exceed its boundaries.

If not, please suggest a revised version.

Reply:
```suggestion

```

Reply:
Can we say:
A collision occurs when a specified position causes an overlay element to exceed its boundary.